CourseGrab/ViewControllers/Home/HomeTableViewHeader.swift

```diff
@@ -20,13 +20,19 @@ class HomeTableViewHeader: UITableViewHeaderFooterView {
         
         titleLabel.font = ._20Semibold
         contentView.addSubview(titleLabel)
+
+        titleLabel.snp.makeConstraints { make in
+            make.leading.trailing.equalToSuperview().inset(20)
+            make.bottom.equalToSuperview().inset(6)
+            make.top.equalToSuperview().inset(6)
+        }
     }
 
     required init?(coder: NSCoder) {
         fatalError("init(coder:) has not been implemented")
     }
 
-    func configure(tableSection: HomeTableViewController.TableSection, section: Int) {
+    func configure(tableSection: HomeTableViewController.TableSection) {
```

Comment:
Change to `configure(for tableSection: ...)` to be more consistent with the other configure functions.